README.md

```diff
@@ -253,6 +253,28 @@ Use the `set_param` method to set new global param values like you would on the
 
 Use the `param?` method to check whether a specific param has been set.  Prefer this over `params.key?` as the task params have special logic for interacting with any runner params that makes the traditional `key?` method unreliable.
 
+##### `before`, `prepend_before`, `after`, `prepend_after`
```

Comment:
@jcredding FYI, I added a README section for these new helper methods (sorry I missed this before you originally reviewed the PR).